### PR TITLE
release-tool: add calendar event for sec-team of container images

### PR DESCRIPTION
**Pull request by kevinwojo**
Adds a calendar event to nudge the security team to review Sourcegraph container image scan results and provide guidance on resolution. This should give enough lead time to prevent a release-blocker situation.

See [RFC 678 for more details](https://docs.google.com/document/d/1v0TXVLPCNA42cQwIYeGLICoy6zfK9KprMRpMa2Fn6IE/edit#).

## Test plan
* CI testing to make sure it doesn't break anything in critical path
* Manual testing by release captain

### CODE REVIEW

File: dev/release/release-config.jsonc [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "previousRelease": "3.39.0",
     "upcomingRelease": "3.39.1",
     // Release dates
+    "oneWorkingWeekBeforeRelease": "18 April 2022 10:00 PST",
```
Comment (kevinwojo) [inline dev/release/release-config.jsonc:19]: The release captain will need to update the next three lines in preparation for the next release (3.40.0)

Comment (unknwon) [reply dev/release/release-config.jsonc:19]: Hmm... I'm completely confused by this file, neither the release captain, release number nor release dates are correct... if we are actually using this file, I'm surprised it is not updated along with other files by the previous captain (likely we did not even know about this?).

**Review by unknwon — APPROVED** [review]
Approve to unblock, if it works the way it is...

File: dev/release/release-config.jsonc (hunk 1 above)
Comment (kevinwojo) [reply dev/release/release-config.jsonc:19]: I think it was updated for the patch release and the release captain forgot to reset it for the next monthly release. We can change that here.

**Pull request merged**